packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -40,16 +44,14 @@ const SearchSuggestions = ({ autoCompleteHits, enterpriseSlug, handleViewAllClic
               <SearchSuggestionItem
                 key={hit.title}
                 url={getLinkToProgram(hit)}
-                /* eslint-disable-next-line no-underscore-dangle */
-                highlightedTitle={hit._highlightResult.title.value}
-                authoringOrganization={hit.authoring_organizations.shift()?.key}
-                title={hit.title}
-                programType={hit.program_type}
+                hit={hit}
+                suggestionItemHandler={handleSubmit}
+                disableSuggestionRedirect={disableSuggestionRedirect}
```

Comment:
hmm looks like you don't actually use this anymore in SearchSuggestionItem? Can it be removed?